packages/components/src/DataTable/DataTable.css

```
@@ -0,0 +1,86 @@
+.dataTableContainer {
+  border: var(--border-base) solid var(--color-border);
+}
+
+.tableContainer {
+  overflow: auto;
+}
+
+.table {
+  width: 100%;
+  border-collapse: collapse;
+  text-align: left;
+}
+
+.table thead tr th {
+  padding: var(--space-base);
+  border-bottom: var(--border-base) solid var(--color-border);
+  white-space: nowrap;
+}
+
+.table thead tr th :first-child {
+  display: flex;
+}
+
+.table tbody tr {
+  border-bottom: var(--border-base) solid var(--color-border);
+}
+
+.table tbody tr:last-child {
+  border-bottom: none;
+}
+
+.table tbody tr td {
+  padding: var(--space-base);
+}
+
+.pinFirstColumn thead tr th:first-child,
+.pinFirstColumn tbody tr td:first-child {
+  position: sticky;
+  left: 0;
+  z-index: 1;
+  height: calc(var(--space-largest) + var(--space-smaller));
+  background-color: var(--color-white);
+}
+
+.pinFirstColumn th:first-child::after,
+.pinFirstColumn td:first-child::after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 0;
+  right: calc(-1 * var(--space-small));
+  width: var(--space-small);
+  height: 100%;
+  background: linear-gradient(
+    90deg,
+    rgba(0, 0, 0, 0.1) 0%,
+    rgba(0, 0, 0, 0) 80%
+  );
+}
+
+.stickyHeader {
+  position: sticky;
+  top: 0px;
+  right: 0;
+  left: 0;
+  z-index: 2;
+  box-shadow: var(--shadow-base);
+  background-color: var(--color-white);
+}
+
+.pinFirstHeaderSortable:first-child:hover {
+  background: var(--color-surface--hover);
+  cursor: pointer;
+}
+
+.sortableColumn:hover {
+  background: var(--color-surface--hover);
+  cursor: pointer;
+}
+
+.clicableRow:hover,
```

typo, would need to fix this across the files that use it

```suggestion
.clickableRow:hover,
```

---

✅